dashboards/google-cloud-composer/cloud-composer-monitor.json

```diff
@@ -0,0 +1,374 @@
+        },
+        "width": 12,
+        "xPos": 0,
+        "yPos": 0
+      },
+      {
+        "height": 3,
+        "widget": {
+          "title": "# Celery Workers",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "legendTemplate": "Active workers",
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_MIN"
+                    },
+                    "filter": "metric.type=\"composer.googleapis.com/environment/num_celery_workers\" resource.type=\"cloud_composer_environment\" resource.label.\"environment_name\"=\"sb-data-projects-composer-load-test\" resource.label.\"location\"=\"us-central1\""
+                  }
+                }
+              },
+              {
+                "legendTemplate": "Min workers",
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_MIN",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_MIN"
+                    },
+                    "filter": "metric.type=\"composer.googleapis.com/environment/worker/min_workers\" resource.type=\"cloud_composer_environment\" resource.label.\"environment_name\"=\"sb-data-projects-composer-load-test\" resource.label.\"location\"=\"us-central1\""
+                  }
+                }
+              },
+              {
+                "legendTemplate": "Max workers",
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_MAX",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_MIN"
+                    },
+                    "filter": "metric.type=\"composer.googleapis.com/environment/worker/max_workers\" resource.type=\"cloud_composer_environment\" resource.label.\"environment_name\"=\"sb-data-projects-composer-load-test\" resource.label.\"location\"=\"us-central1\""
```

Comment:
> resource.label.\"environment_name\"=\"sb-data-projects-composer-load-test\" resource.label.\"location\"=\"us-central1\"

These filters should not exist... they only apply to your project but wouldn't work when a user tries to use this on their own cluster

Reply:
Same goes for Min workers and Active workers